arch/arm/src/stm32h7/stm32_usbhost.c

```diff
@@ -0,0 +1,245 @@
+/****************************************************************************
+ * arch/arm/src/stm32h7/stm32_usbhost.c
```

Comment:
@btashton - if you elongate the "ruler" `***` the long line warning  will go away.

Reply:
I wanted to keep the correct bounds for the rest, but I guess this is the way for now.  What I just pushed should pass.